php-nginx/entrypoint.sh

```diff
@@ -116,9 +116,7 @@ ${PHP_DIR}/bin/php /whitelist_functions.php
 
 if [ -f "${APP_DIR}/composer.json" ]; then
     # run the composer scripts for post-deploy
-    COMPOSER_CMD="env PATH=${PATH}:${PHP_DIR}/bin ${PHP_DIR}/bin/php \
-    /usr/local/bin/composer \
-    --no-ansi"
+    COMPOSER_CMD="php /usr/local/bin/composer --no-ansi"
```

Comment:
nit: I think we can just call this directly w/o setting `COMPOSER_CMD` now, i.e. 

```sh
if su www-data -c php /usr/local/bin/composer --no-ansi run-script -l" \
# ...
```